website/docs/language/functions/regex.html.md

```diff
@@ -73,7 +73,7 @@ of the pattern must be escaped as `\\`.
 | `\PN`          | The opposite of `\pN`                                                            |
 | `\P{Greek}`    | The opposite of `\p{Greek}`                                                      |
 | `xy`           | `x` followed immediately by `y`                                                  |
-| `x|y`          | either `x` or `y`, preferring `x`                                                |
+| `x\|y`         | either `x` or `y`, preferring `x`                                                |
```

Comment:
This works!

```suggestion
| <code>x&#124;y</code> | either `x` or `y`, preferring `x`                                         |
```

<img width="442" alt="Screen Shot 2021-02-08 at 10 40 54 AM" src="https://user-images.githubusercontent.com/6210214/107242582-20ebd900-69fa-11eb-9d51-fd9dc9b7b320.png">

Reply:
awesome find! thanks for the help!